tests/e2e/plugins/module-setup-analytics.php

```diff
@@ -337,6 +272,31 @@ function () {
 			true
 		);
 
+		register_rest_route(
+			REST_Routes::REST_ROOT,
+			'modules/analytics-4/data/container-destinations',
+			array(
+				'methods'             => 'GET',
+				'callback'            => function ( \WP_REST_Request $request ) use ( $ga4_webdatastreams ) {
```

Comment:
`ga4_webdatastreams` is not used:

```suggestion
				'callback'            => function ( \WP_REST_Request $request ) {
```